### ISSUE TITLE
Accessibility Expansion + Animation

### ISSUE THREAD

**Issue by mauii14** [thedaviddias/Front-End-Checklist#81]
## Accessibility section should include 

 **Title Tags and Image Tags** 
   - provide text to screen readers.  This is important for developers to understand. 
- Titale tags also tell screen readers where the link goes

**Color Contrast**
- this section should include links to multiple palette tools, including
-  Paletton
-   Palette.io

**Colors should not be hard-coded**
`color: #white ;`
this is hard-coded, and difficult for users to override
`color: #fff;`
this is correct 

**Keyboard Section** 
- use an incorporate the keyboard instructions from whatever backend framework is used
- Skip Menu should also be included

I also agree that an Animation section should be included. 
**Animation**
-  flashing elements should not be used for users with epilepsy. It may cause seizures.

**Comment by thedaviddias** [comment]
Hi @mauii14, thanks for all these elements! Don't hesitate to do a PR, but in all cases, I'll analyze and add your requests in the next hours.

(don't know how I forgot to mention the essential "skip menu" 😱)

**Comment by thedaviddias** [comment]
@mauii14, I was thinking about how to keep the checklist "short" without putting all the items necessary, but enough to ensure accessibility... Maybe we can have an item saying "Check if you respect all accessibility rules" and put some links to tools that will list all possible issues.

Do you see what I mean?
